lib/brexit_checker/change_notes.yaml

```diff
@@ -42,3 +42,7 @@ change_notes:
      action_id: T064
      note: "Link changed to provide hauliers with updated guidance."
      date: 2019-10-02
+   - uuid: "7683bdd6-eef4-4d78-b78f-9e507e0ba0da"
+     type: addition
+     action_id: T093
+     date: 2019-10-03
```

Comment:
Spotted afterwards: we should update this to today's date.

Reply:
on it, will have to change this on all these batch of notifications! Whoops!
Thanks for spotting.

Reply:
done